Userland/wc.cpp

```diff
@@ -2,143 +2,131 @@
 #include <AK/Vector.h>
 #include <LibCore/CArgsParser.h>
 
-#include <ctype.h>
 #include <stdio.h>
+#include <sys/stat.h>
 
-static bool output_chars = false;
-static bool output_words = false;
-static bool output_lines = false;
+struct WhatToPrint {
+    bool line_flag=false;
+    bool byte_flag=false;
+    bool char_flag=false;
+    bool word_flag=false;
+};
 
-struct Count {
-    String file;
-    unsigned long chars = 0;
-    unsigned long words = 0;
-    unsigned long lines = 0;
+struct Word {
```

Comment:
Why is this in a struct? Why is it called `Word`? It doesn't represent a word.

Reply:
There was not need to be a struct, I was just a bit confused originally by what you meant by C pretending to be C++ so I just tried some random things out. It definitely was not my brightest move. As for it being called word, that was just a poor naming choice.